File: src/main/java/no/nav/fo/veilarbregistrering/domain/Veileder.java
package no.nav.fo.veilarbregistrering.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;
import lombok.experimental.Accessors;
import no.nav.fo.veilarbregistrering.orgenhet.NavEnhet;

@Data
@Accessors(chain = true)
@ToString
@EqualsAndHashCode
public class Veileder {
    String ident;
    NavEnhet enhet;
}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/registrering/veileder/Veileder.kt
package no.nav.fo.veilarbregistrering.registrering.veileder

import no.nav.fo.veilarbregistrering.orgenhet.NavEnhet

data class Veileder(var ident: String? = null, var enhet: NavEnhet? = null) {
    override fun toString(): String = "Veileder(ident=$ident, enhet=$enhet)"
}